pkg/utils/clusterset/clusterset.go

```diff
@@ -76,3 +82,112 @@ func GenerateClustersetSubjects(cache *cache.AuthCache) map[string][]rbacv1.Subj
 
 	return clustersetToSubjects
 }
+
+// BuildAdminRoleRules builds the clustesetadminroles
+func BuildAdminRole(clustersetName, clusteroleName string) *rbacv1.ClusterRole {
```

Comment:
more comments will be helpful to describe the permission for each clusterrole

Reply:
done